wikicharts/content_gap_plots.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        df = pd.read_csv("../metrics/geo_data_metrics.tsv", sep = '\t')
</pre><p>Should use the parameter you added to parameters.py.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>